.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,49 @@
+<!--
+Thanks for interest in Openshift.io and apologies for making your experience not yet as awesome as we strive for.  
+
+Please follow template provided below to report a bug you have encountered and will make sure to improve it.
```

Comment:
`and will make sure to improve it` has a typo. It needs a `we` in it. `and we will make sure to improve it`